docs/source/frameworks/picklable.rst

```diff
@@ -2,41 +2,46 @@
 Picklable Model
 ===============
 
-Here's an example of saving any Python object or function as model, and create a runner
-instance:
+For custom ML models created with pure Python code, a simple way to make it work with
+the BentoML workflow is via ``bentoml.picklable_model``.
```

Comment:
```suggestion
For custom ML models created with plain Python code, BentoML provides ``bentoml.picklable_model`` as a solution to incorporate any arbitrary Python functions with BentoML’s serving workflow.
```

Reply:
this is not exactly accurate. not just function but also custom model classes

Reply:
btw I will come back and revisit this doc soon, this is just a quick first version